src/component/GenreFilter.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import {connect} from 'react-redux';
+import Paper from '@material-ui/core/Paper';
+import Button from '@material-ui/core/Button';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+
+import Checkbox from '@material-ui/core/Checkbox';
+import { getAllGenres, putSelectedGenre, deleteSelectedGenre,getMoviesByPage } from '../store/actions/MovieActions';
+class GenreFilter extends React.Component{
+    
+    componentDidMount(){
+        if (this.props.genres.length == 0){
+            this.props.getAllGenres();
+            console.log("genres", this.props.genres)
+        }
+    }
+    handleChange = (event) =>{
+        if (event.target.checked){
+            this.props.putSelectedGenre(event.target.value);
+        }else{
+            this.props.deleteSelectedGenre(event.target.value);
+        }
+    }
+    handleFilters = ()=>{
+        this.props.getMoviesByPage({page: 0/10, perPage:10, genreFilter: this.props.selectedGenres});
```

Comment:
` 0/10` cemu ovo ?

Reply:
Greska, treba da stoji samo 0 i to znaci da kad se primeni filter da ga vrati na prvu stranicu